src/Runner.Worker/Container/DockerCommandManager.cs

```diff
@@ -197,7 +197,8 @@ public async Task<int> DockerRun(IExecutionContext context, ContainerInfo contai
 
             dockerOptions.Add($"--workdir {container.ContainerWorkDirectory}");
             dockerOptions.Add($"--rm");
-
+            dockerOptions.Add($"-t");
```

Comment:
you might want to add the `-t` to where the runner run `docker exec` as well.

Reply:
in the public method here?

https://github.com/actions/runner/blob/6e2fa11b267e79fb553b1b759fbe041eb4db945f/src/Runner.Worker/Container/DockerCommandManager.cs#L301

Reply:
I added it for you. 😄

Reply:
Should we use "-it" in both places?

Reply:
Thanks all